src/utils/raw.rs

```diff
@@ -41,7 +41,7 @@ impl RawWriter {
             rgb.format = rgb::Format::Rgba;
             rgb.depth = 16;
             //rgb.depth = 8;
-            rgb.alpha_premultiplied = true;
+            rgb.premultiply_alpha = true;
```

Comment:
leftover code at the line above this?

Reply:
yeah this whole file is sort of a place holder that's not really used. i change it periodically for testing.